concordium-node/Cargo.lock

```diff
@@ -1179,7 +1179,6 @@ dependencies = [
  "rand 0.7.3",
  "random_oracle",
  "serde 1.0.145",
- "wasm-bindgen",
```

Comment:
I didn't make any changes in `Cargo.toml`, so I assume we just forgot to push the lock file recently.

Reply:
It might be connected to the conflicts in base.
Try updating it and update the lock file

Reply:
I think the lock files are up-to-date as long as your base is. It looks like maybe your base is not up-to-date?

This was changed recently (like yesterday).